rcgen/CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 
 # Changes
 
-## Unreleased
+## Release 0.12.0 - December 16, 2023
 
-- Rename `RcgenError` to `Error` to avoid stuttering when used fully-qualified via `rcgen::`.
-- Upgrade to `ring` `v0.17`.
+- Rename `RcgenError` to `Error`. Contributed by [thomaseizinger](https://github.com/thomaseizinger).
+- Lock down the `From` impls for `Error` and make it future-proof.
```

Comment:
I think we can give more information here. I would also try to put the user-facing changes up-front and then describe the rationale. What about:

```suggestion
- `Error::PemError` has been changed to hold a `String` value. The `From<_>` traits for external error types have been removed. This allows rcgen to update dependent crates without impacting API consumers.
```